counsel.el

```diff
@@ -731,6 +731,25 @@ With a prefix arg, restrict list to variables defined using
       (when doc
         (lv-delete-window)))))
 
+;;** `counsel-tab-bar'
+(defun counsel--tab-name (tab)
+  "Return a name for a tab."
+  (cdr (car (cdr tab))))
```

Comment:
This is assuming that `name` is the first element of a tab's alist, but alists are unordered collections. You should instead write:
```el
(cdr (assq 'name tab))
```